selenpy/support/browser.py

```diff
@@ -43,3 +47,13 @@ def wait_until(webdriver_condition, timeout=None, polling=None):
         polling = config.poll_during_waits
 
     return wait_for(get_driver(), webdriver_condition, timeout, polling)
+
+def switch_to_alert():
+    try:
+        WebDriverWait(get_driver(),5).until(EC.alert_is_present(),'Timed out waiting for alerts to appear')
+        return get_driver().switch_to.alert
```

Comment:
Why we need to return here?

Reply:
if not return, when using switch_to_alert().text to get text of alert will display error: `AttributeError: 'NoneType' object has no attribute 'text'`